src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/NamespacesWfsTest.java

```diff
@@ -176,9 +181,17 @@ public void virtualServiceStoredQueryNamespacesGml31() throws Exception {
         } finally {
             storedQueryProvider.removeAll();
             assertTrue(storedQueryProvider.listStoredQueries().size() == 1);
+            configurePerWorkspaceStoredQueries(true);
         }
     }
 
+    private void configurePerWorkspaceStoredQueries(boolean activate) {
```

Comment:
Why is this enabled? I don't see new app-schema tests related to stored queries. Makes me worry there is a backwards incompatibility related reason somehow...

Reply:
Good catch, indeed backward compatibility should support this use case, changes applied.